app/forms/openid_connect_authorize_form.rb

```diff
@@ -199,10 +199,6 @@ def ial
     Saml::Idp::Constants::AUTHN_CONTEXT_CLASSREF_TO_IAL[ial_values.sort.max]
   end
 
-  def aal
-    Saml::Idp::Constants::AUTHN_CONTEXT_CLASSREF_TO_AAL[aal_values.sort.max]
-  end
```

Comment:
this is very surprising to me that it's not used...? are we getting aal elsewhere?

Reply:
Looks like the usage was removed in https://github.com/18F/identity-idp/pull/3921/files#diff-4bb1f0f69bba775a3294d2476250049f71212572f59eb8053786131eafb905c9L64-L66